### PR TITLE
plotkicadsch version 0.2.0

### PR DESCRIPTION
**Pull request by jnavila**

### CODE REVIEW

File: packages/kicadsch/kicadsch.0.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,17 @@
+opam-version: "1.2"
+maintainer: "Jean-Noel Avila <jn.avila@free.fr>"
+author: "Jean-Noel Avila <jn.avila@free.fr>"
+homepage: "https://jnavila.github.io/plotkicadsch/"
+bug-reports: "https://github.com/jnavila/plotkicadsch/issues"
+license: "ISC"
+dev-repo: "https://github.com/jnavila/plotkicadsch.git"
+build: [
+  [ "jbuilder" "subst" "-n" "kicadsch" ] {pinned}
```
Comment (rgrinberg) [inline packages/kicadsch/kicadsch.0.2.0/opam]: You may use the `name` variable rather than the hard coded package name here as well.

Comment (jnavila) [reply packages/kicadsch/kicadsch.0.2.0/opam]: Thanks for the comment. BTW the compilation failure is strange. Patience_diff is required in version v0.10.0 (in the opam file) but the CI keeps on using v0.9.0. Moreover, v0.10.0 is available in opam but is not present in this repo. 

I dont understand how all this works so I have no clue on how to solve this issue.

**Comment by jpdeplaix** [comment]
The errors popping in the CIs are due to the fact that janestreet packages are versionned slightly differently than other packages due to a change in their versionning scheme.
To fix this issue please replace ```"patience_diff" {>= "0.10.0"}``` by ```"patience_diff" {>= "v0.10.0"}``` (notice the ```v``` prefix).

To read a bit more on this issue: https://github.com/dbuenzli/topkg/issues/122

**Comment by jpdeplaix** [comment]
I'm not sure why opam2 on 4.06.0 fails with:
```
[WARNING] Upgrade is not possible because of conflicts or packages that are no longer available:
The following dependencies are the cause:
  - ocaml -> ocaml-base-compiler >= 4.06.0
    no matching version
  - ocaml -> ocaml-system >= 4.06.0
    no matching version
  - ocaml -> ocaml-variants < 4.06.1~
    conflict with the base packages of this switch
```
With opam1 it fails with:
```
No package matches patience_diff>=v0.10.0
```
but anyway there is definitely something wrong here (probably not on your part). I will investigate.

cc @trefis just in case

**Comment by jpdeplaix** [comment]
Ok, just restarted the CI but still the same. Moreover it works with Travis.
@avsm what are the differences between the tools installed with Travis and on the Mirage-CI ? I'm confused :/

**Comment by avsm** [comment]
One of them is opam1, and the other is opam2.  Datakit-CI tests both opam1 and opam2 and it looks like only opam2 is failing here?

**Comment by jpdeplaix** [comment]
No, both are failling but only one detects it as an actual error, the other one silently fails because "package not available"

**Comment by jnavila** [comment]
Can I test anything to help?

**Comment by jpdeplaix** [comment]
I'll do more checks locally today. I'll merge it by the end of the day anyway as it seems to work on Travis.

**Comment by jpdeplaix** [comment]
Ok, found the problem (*) ! Hopefully it will be fixed soon enough.

(*) @avsm it's mirage-ci. I'll get on it.

**Comment by jpdeplaix** [comment]
@jnavila I'm still finding a way to fix the issue properly (I have to make sense of some bits of code first). In the meantime, could you just rebase your branch over ocaml/opam-repository master ? The issue comes from the fact that mirage-ci (the one that fails) just use your branch and doesn't merge it against master. So of course with just your branch ```patience_diff.v0.10.0``` doesn't exist.

**Comment by jnavila** [comment]
still failing 😢

**Comment by jpdeplaix** [comment]
it seems that your last commit was somehow eliminated ?! Putting back the ```v``` prefix to ```0.10.0``` should fix the issue.

**Comment by jnavila** [comment]
True. A peril of the force push when dealing with several working copies.

**Comment by jpdeplaix** [comment]
Merging. The Alpine failure on with opam1.2 should be fixed by #11312

**Pull request merged**